packages/shared/forks/ReactFeatureFlags.native-oss.js

```diff
@@ -10,7 +10,7 @@
 import typeof * as FeatureFlagsType from 'shared/ReactFeatureFlags';
 import typeof * as ExportsType from './ReactFeatureFlags.native-oss';
 
-export const debugRenderPhaseSideEffectsForStrictMode = false;
+export const debugRenderPhaseSideEffectsForStrictMode = true;
```

Comment:
Double logging was disabled for React Native OSS. Was this decision intentional?

Reply:
Maybe we should separate this change from the rest of the PR?